cli/src/pcluster/models/cluster.py

```diff
@@ -368,11 +368,12 @@ def create(
                 validator_suppressors, validation_failure_level
             )
 
-            LOGGER.info("Generating artifact dir and uploading config...")
+            LOGGER.info("Generating artifact dir and uploading config and instance types data...")
```

Comment:
Nit Pick: Change the log message to something like "Generating artifact dir, uploading config and instance types data..."

Reply:
fixed